packages/notion-client/src/notion-api.ts

```diff
@@ -332,6 +332,18 @@ export class NotionAPI {
       collectionView?.format?.board_columns_by ||
       collectionView?.format?.collection_group_by
 
+    let filters = []
+    if (collectionView.format?.property_filters) {
+      filters = collectionView.format?.property_filters.map((filterObj) => {
+        // console.log('map filter', filterObj)
+        //get the inner filter
+        return {
+          filter: filterObj.filter.filter,
```

Comment:
Let's make sure to use `?.` everywhere here just to make sure the code doesn't crash. I generally don't trust notion's data model to be consistent across future updates.

Reply:
I don't trust their model either. 😁
Good thinking. updated this.